### PR TITLE
feat: iis example config for google cloud exporter

### PR DESCRIPTION
**Pull request by algchoo**
### Proposed Change

Added example configuration using iis receiver and exporting to google cloud
* tested configuration on Windows Server 2019

##### Checklist
- [x] Changes are tested
- [ ] CI has passed

### CODE REVIEW

File: config/google_cloud_exporter/iis/config.yaml [hunk 1]
```diff
@@ -0,0 +1,30 @@
+receivers:
+  iis:
+    collection_interval: 10s
```
Comment (jsirianni) [inline config/google_cloud_exporter/iis/config.yaml:3]: Lets be consistent with 60s collection interval. I believe the Google minimum is 30 seconds.

File: config/google_cloud_exporter/iis/config.yaml [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+processors:
```
Comment (jsirianni) [inline config/google_cloud_exporter/iis/config.yaml:5]: We probably need resource detection, resource, and resource attributes transposer here. I am curious to see how the metrics look into the metric explorer.

**Review by jsirianni — CHANGES_REQUESTED** [review]

File: config/google_cloud_exporter/iis/config.yaml [hunk 3]
```diff
@@ -0,0 +1,30 @@
...
+
+exporters:
+  googlecloud:
+    retry_on_failure:
+      enabled: false
+    metric:
+      prefix: workload.googleapis.com
+    resource_mappings:
+      - source_type: ""
+        target_type: generic_node
```
Comment (BinaryFissionGames) [inline config/google_cloud_exporter/iis/config.yaml:19]: Now that v0.6.0 is released, we can simplify these a LOT. The exporter now has:
- Built in normalize sums processor (doesn't need to be configured separately)
- Built in Batch processor (doesn't need to be configured separately)
- Automatic mapping to generic_node
- metric prefix is default set to "workload.googleapis.com"
- retry_on_failure is default disabled

I think from now on, we should try using the new exporter defaults with these configs.

Let me know if any of that was unclear.

https://github.com/observIQ/observiq-otel-collector/tree/main/exporter/googlecloudexporter

File: config/google_cloud_exporter/iis/config.yaml (hunk 2 above)
Comment (jsirianni) [reply config/google_cloud_exporter/iis/config.yaml:5]: See Brandon's comment

**Review by cpheps — APPROVED** [review]

**Review by BinaryFissionGames — APPROVED** [review]

**Review by jsirianni — APPROVED** [review]

**Pull request merged**